webui.go

```diff
@@ -170,9 +170,9 @@ func strTicketNumCsv(height int64, all, own int) string {
 
 // strPurchasedCsv converts data relating to number of tickets purchased this
 // block to an easy-to-write string for the csv files.
-func strPurchasedCsv(height int64, purchased int) string {
+func strPurchasedCsv(height int64, purchased, leftWindow int) string {
 	var buf bytes.Buffer
-	buf.WriteString(fmt.Sprintf("%v,Purchased,%v\n", height, purchased))
+	buf.WriteString(fmt.Sprintf("%v,Purchased,%v,%v\n", height, purchased, leftWindow))
```

Comment:
Don't inject this as an extra field in purchased, add another string into the csv file eg

buf.WriteString(fmt.Sprintf("%v,RemainingToBuy,%v\n", height, leftWindow))